code/game/machinery/launch_pad.dm

```diff
@@ -209,18 +209,29 @@
 	resistance_flags = FLAMMABLE
 	max_integrity = 150
 	var/obj/machinery/launchpad/briefcase/pad
+	var/folder_path = /obj/item/folder //this is the path of the folder that gets spawned in New()
 
-/obj/item/briefcase_launchpad/Initialize()
+/obj/item/storage/briefcase_launchpad/Initialize()
 	. = ..()
 	pad = new(src)
 
-/obj/item/briefcase_launchpad/Destroy()
+/obj/item/storage/briefcase_launchpad/Destroy()
 	if(!QDELETED(pad))
 		qdel(pad)
 	pad = null
 	return ..()
 
-/obj/item/briefcase_launchpad/attack_self(mob/user)
+/obj/item/storage/briefcase_launchpad/ComponentInitialize()
+	. = ..()
+	GET_COMPONENT(STR, /datum/component/storage)
+	STR.max_w_class = WEIGHT_CLASS_NORMAL
+	STR.max_combined_w_class = 21
```

Comment:
Do you need to retype it as a child of storage if you can just add the storage component?

Reply:
or you could go in the opposite direction instead and make it a child of real briefcases

Reply:
Making it a child of real briefcases is even better, good idea